projects/ion/src/lib/core/types/card.ts

```diff
@@ -19,6 +19,7 @@ interface Header {
   buttons?: ButtonBase[];
   chips?: IonChipProps[];
   icon?: IconType;
+  infoTooltip?: SafeAny;
```

Comment:
Couldn't be typed as `TooltipProps`?

Reply:
because TooltipProps only takes three arguments: the ionTooltipTitle, ionTooltipColorScheme and the ionTooltipPosition. And these arguments are passed plus three others: the ionTooltipArrowPointAtCenter, ionTooltipTrigger and the ionTooltipShowDelay

Reply:
I think that you can add those types on TooltipProps as optional arguments, since they're correctly typed, and use it there 😄

Reply:
Alright, I'm going to put these optional arguments in the TooltipProps.